lib/cpp/hawkes/simulation/simu_hawkes.cpp

```diff
@@ -56,15 +53,13 @@ bool Hawkes::update_time_shift_(double delay, ArrayDouble &intensity,
 void Hawkes::reset() {
   for (unsigned int i = 0; i < n_nodes; i++) {
     for (unsigned int j = 0; j < n_nodes; j++) {
-      if (kernels[i * n_nodes + j] != nullptr)
-        kernels[i * n_nodes + j]->rewind();
+      if (kernels[i * n_nodes + j] != nullptr) kernels[i * n_nodes + j]->rewind();
```

Comment:
I know it was `unsigned int` before, I would generally prefer `std::uint32_t` or similar

Reply:
Understood...
I will open a new PR about that because it is not related to `HawkesCumulant` or tensorflow